Modix.Data/Models/Core/ClaimMappingSearchCriteria.cs

```diff
@@ -14,7 +14,7 @@ public class ClaimMappingSearchCriteria
         /// <summary>
         /// A set of <see cref="ClaimMappingEntity.Type"/> values, defining the <see cref="ClaimMappingEntity"/> entities to be returned.
         /// </summary>
-        public IReadOnlyCollection<ClaimMappingType>? Types { get; set; }
+        public ClaimMappingType[]? Types { get; set; }
```

Comment:
Why? (multiple occurences) 🤔

Reply:
From my post above:
> Switched IReadOnlyCollection to T[] in search criteria models because npgsql is currently running into errors translating non-arrays into SQL.

Reply:
missed that, my bad